### PR TITLE
Development of vertcoin.io website

### PR DESCRIPTION
**Pull request by docccjr**
**The greatest marketing resource for Vertcoin ever**
The brand new website vertcoin.io will be the GREATEST marketing resource for Vertcoin ever, but it is not free to develope and maintain. The website is live and has been live for over 1 year so this is an ongoing project. I have expenses on the domain, server, licenses + time spent. The goal is to do regular maintenance and development on the website and add new content, features, functonality and do some heavy search engine optimization - AND SPREAD THE WORD!

**What has been done so far**

- Created the whole website with coded design. The framework is Wordpress.
- Made it easy to use on mobilephones.
- made vertcoin.io multilangual, 11 languages. The website adopts to the language in your browser.
- added mining statistics: https://vertcoin.io/mining-statistics/
- added multiple wallets and a tiny review of them: https://vertcoin.io/download-wallet/
- started on the blog: https://vertcoin.io/blog/
- updated history: https://vertcoin.io/history/
- Google Analytics and Search Console has been installed.

**What will I do every month**

- Add new valuable content.
- Add new functionality such as Mining Statistics.
- improve the user experience of the website.
- Do heavy SEO work and focus on specific keywords. Such as: "Mine crypto with my computer".
- Post updates on Discord / Reddit on the performance of the website. Number of vistors etc.
- Take feedback from the community and do changes based on that.
- Talk with marketing team and dev team regarding important updates on the website.

**Who am I**
I am a web developer located in Norway. I have a team of 7 people so vertcoin.io will never die or be stopped working on.

**Why am I doing this**
I love Vertcoin and web development, but I do not want to lose money on this project, especially when the project is value-creating and will lead to vertcoin becoming more valuable.

**Costs**
I am looking for a total of 5000 VTC.
2500 will go to cover basic costs such as the domain, server, licenses.

2500 will go towards the work already done + what we will do. These coins will be distributed to my team so they get the motivation to do the "little extra" for vertcoin.io.

Sorry for spelling mistakes. English is my 2nd language.

Thank you for reading :)

### CODE REVIEW

**Comment by vertiond** [comment]
No mention of cost / funding required in description. Please update the commit as well

File: new-vertcoin-website-io.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: fr
 title: Development of vertcion.io website
 author: docccjr
 date: january 6, 2023
-amount: 5000
+amount: 5000 VTC
```
Comment (vertiond) [inline new-vertcoin-website-io.md]: Do not add anything other than a number here

**Review by docccjr — COMMENTED** [review]
Added text

**Comment by docccjr** [comment]
Updated with costs

**Comment by vertiond** [comment]
CCS proposals are now taking place at https://github.com/vertcoin-project/ccs-proposals/ 
Feel free to open there - this PR will now be closed.

**Pull request closed without merging**